go.mod

```diff
@@ -110,6 +110,8 @@ require (
 	github.com/spf13/pflag v1.0.5
 	github.com/tchap/go-patricia v2.3.0+incompatible // indirect
 	github.com/urfave/cli v1.22.2
+	// e694b7bb0875 is v3.4.7
```

Comment:
consider upgrade to etcd-v3.4.9 because some bugfix (eg. https://github.com/etcd-io/etcd/pull/11888 & https://github.com/etcd-io/etcd/pull/11652)?

Reply:
Updating now